connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/UCSingleCatalog.scala

```diff
@@ -48,7 +49,19 @@ class UCSingleCatalog extends TableCatalog with SupportsNamespaces {
       columns: Array[Column],
       partitions: Array[Transform],
       properties: util.Map[String, String]): Table = {
-    deltaCatalog.createTable(ident, columns, partitions, properties)
+    val isExternal = properties.containsKey(TableCatalog.PROP_EXTERNAL)
+    def isPathTable = ident.namespace().length == 1 && new Path(ident.name()).isAbsolute
+    if (!isExternal && !properties.containsKey(TableCatalog.PROP_LOCATION) && !isPathTable) {
+      val newProps = new util.HashMap[String, String]
+      newProps.putAll(properties)
+      // TODO: here we use a fake location for managed table, we should generate table location
+      //       properly when Unity Catalog supports creating managed table.
+      newProps.put(TableCatalog.PROP_LOCATION, "file:///tmp/fake")
+      newProps.put(TableCatalog.PROP_IS_MANAGED_LOCATION, "true")
```

Comment:
Why are we passing `TableCatalog.PROP_IS_MANAGED_LOCATION -> true` instead of `TableCatalog.PROP_EXTERNAL -> false` to the underlying DeltaCatalog?

Reply:
if these 2 variables are mutually exclusive to each other, then we should have an assert here as well as in Delta code that we never set them in inconsistently

Reply:
We should respect the definition from Spark. The `PROP_EXTERNAL` should only be true if the CREATE TABLE command has the keyword `EXTERNAL` specified. For example, `CREATE EXTERNAL TABLE t ...`. It should not be true even if the keyword `LOCATION` is specified like `CREATE TABLE t ... LOCATION ...`, although most catalogs will create this table as external table since it has a user-specified location.

On the other hand, `PROP_IS_MANAGED_LOCATION` is used to indicate that the location is not user-specified by managed by the system, which is exactly the case here.

Reply:
that said, these two properties are not mutually exclusive to each other. @tdas

Reply:
I can add comments here to repeat the Spark definition for `PROP_IS_MANAGED_LOCATION`.